packages/taquito-michel-codec/src/michelson-types.ts

```diff
@@ -48,16 +48,18 @@ export type MichelsonInstruction =
     InstrX<"EMPTY_BIG_MAP", [MichelsonType, MichelsonType]> |
     InstrX<"LAMBDA", [MichelsonType, MichelsonType, InstructionList]> |
     InstrX<"DIP", [IntLiteral, InstructionList] | [InstructionList]> |
+    InstrX<"VIEW", [StringLiteral, MichelsonType]> |
     InstrPrim<"DROP" | "PAIR" | "UNPAIR" | "DUP" | "GET" | "UPDATE", [IntLiteral]>;
 
 // Types
 
 export type MichelsonSimpleComparableTypeID = "string" | "nat" | "int" | "bytes" | "bool" | "mutez" |
-    "key_hash" | "address" | "timestamp" | "never" | "key" | "unit" | "signature" | "chain_id";
+"key_hash" | "address" | "timestamp" | "never" | "key" | "unit" | "signature" | "chain_id";
 
 export type MichelsonTypeID = MichelsonSimpleComparableTypeID |
-    "option" | "list" | "set" | "contract" | "operation" | "pair" | "or" | "lambda" | "map" | "big_map" |
-    "sapling_transaction" | "sapling_state" | "ticket" | "bls12_381_g1" | "bls12_381_g2" | "bls12_381_fr";
+"option" | "list" | "set" | "contract" | "operation" | "pair" | "or" | "lambda" | "map" | "big_map" |
+"sapling_transaction" | "sapling_state" | "ticket" | "bls12_381_g1" | "bls12_381_g2" | "bls12_381_fr" |
+"chest_key" | "chest";
```

Comment:
Why do "chest_key" and "chest" are not part of the MichelsonType (at line 107)?

Reply:
My bad! It wasn't referenced this was as a subtype of MichelsonType anywhere in the code so it didn't cause an error. `chest` and `chest_key` are things in themselves without literal Michelson representation. They can be UNPACK'ed or given as arguments only. This is why I missed it. Thank you.